data/tools/wesnoth_addon_manager

```diff
@@ -292,24 +302,34 @@ if __name__ == "__main__":
             args.upload = args.upload.rstrip("/")
 
             # New style with _server.pbl
-            pblfile = os.path.join(args.upload, "_server.pbl")
+            pbl_file_name = os.path.join(args.upload, "_server.pbl")
             name = os.path.basename(args.upload)
             wmldir = args.upload
             cfgfile = None # _main.cfg will be uploaded with the rest
             ignfile = os.path.join(args.upload, "_server.ign")
         else:
             # Old style with external .pbl file
-            pblfile = args.upload
+            pbl_file_name = args.upload
             name = os.path.basename(args.upload)
             name = os.path.splitext(name)[0]
             wmldir = os.path.join(os.path.dirname(args.upload), name)
             cfgfile = args.upload.replace(".pbl", ".cfg")
             ignfile = args.upload.replace(".pbl", ".ign")
 
         if args.pbl:
-            pblfile = args.pbl
+            pbl_file_name = args.pbl
+
+        with open(pbl_file_name, 'r') as pbl_file:
+            pbl_text = pbl_file.read()
+
+        for key_value_pair in args.pbl_key or []:
+            key = key_value_pair[0]
+            value = key_value_pair[1]
+            if not re.match("^[a-zA-Z]+$", key):
+                raise ValueError("non-standard --pbl-key " + key)
+            pbl_text = pbl_text + '\n' + key + '="' + value.replace('"', '\\"') + '"'
```

Comment:
Are you sure this is correct? Usually in WML double quotes are escaped by doubling them. (" -> "")

Reply:
good catch! Indeed, there was a mistake here. Fixed.